orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/ScaleToClusterResizeStrategy.groovy

```diff
@@ -73,6 +75,6 @@ class ScaleToClusterResizeStrategy implements ResizeStrategy{
     capacity.desired = Math.max(capacity.min, capacity.desired)
     capacity.desired = Math.min(capacity.desired + increment, capacity.max)
 
-    return capacity
+    return new CapacitySet(null, capacity)
```

Comment:
This strategy doesn't take the capacity from a single server group (rather it looks for the max across every server group in a cluster) ... as such there is no suitable original capacity to capture.